src/guide/transitions.md

```diff
@@ -35,7 +35,10 @@ When an element with transition is inserted or removed, Vue will:
 A typical CSS transition looks like this:
 
 ``` html
-<div v-if="show" transition="expand">hello</div>
+<div id="app">
+    <div v-if="show" transition="expand">hello</div>
+    <button @click="toggle">botao</button>
```

Comment:
for beginners, yes

2016-02-16 15:47 GMT-02:00 Evan You <notifications@github.com>:

> In src/guide/transitions.md
> <https://github.com/vuejs/vuejs.org/pull/237#discussion_r53048528>:
>
> > @@ -35,7 +35,10 @@ When an element with transition is inserted or removed, Vue will:
> >  A typical CSS transition looks like this:
> >
> >  ``` html
> > -<div v-if="show" transition="expand">hello</div>
> > +<div id="app">
> > +    <div v-if="show" transition="expand">hello</div>
> > +    <button @click="toggle">botao</button>
>
> I mean, we should probably use toggle here so everyone can understand?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vuejs/vuejs.org/pull/237/files#r53048528>.
>